src/toil/jobStores/aws/utils.py

```diff
@@ -143,10 +143,41 @@ def binaryToAttributes(cls, binary):
         assert len(encoded) <= cls._maxEncodedSize()
         n = cls.maxValueSize
         chunks = (encoded[i:i + n] for i in range(0, len(encoded), n))
-        attributes = {cls._chunkName(i): chunk for i, chunk in enumerate(chunks)}
-        attributes.update({'numChunks': len(attributes)})
+        attributes = {cls._chunkName(i): chunk.decode("utf-8") for i, chunk in enumerate(chunks)}
+        attributes.update({'numChunks': str(len(attributes))})
         return attributes
 
+    @classmethod
+    def attributeDictToBoto3(cls, attributes: Dict[str, str]) -> List[AttributeTypeDef]:
```

Comment:
Since we no longer have Boto 2 and Boto 3, maybe this should be called `attributeDictToList` or something?